pandas/_libs/tslibs/timezones.pyx

```diff
@@ -314,3 +314,21 @@ cpdef bint tz_compare(object start, object end):
     """
     # GH 18523
     return get_timezone(start) == get_timezone(end)
+
+
+cpdef tz_normalize(object tz):
```

Comment:
this is not a great name, normalize has a specific meaning to timestamps. maybe ``tz_standardize``?